src/schema/feeds.ts

```diff
@@ -230,6 +230,46 @@ export const typeDefs = /* GraphQL */ `
       supportedTypes: [String!]
     ): PostConnection! @auth
 
+    """
+    Get feed preview
+    """
+    feedPreview(
```

Comment:
We have two options here:
* Use the approach as you present here, where we use the user's tags from the db for the feed
* Accept the tags from the client and forward them downstream to the feed

Reply:
Since the whole flow of the frontend with tags selection is to save the items to user settings it was natural to just load them. Like that we can use existing generators and thats how it was resolved in DR.

Only place where I see the need to forward tags from frontend is that if we want to allow anonymous user to preview feed. Since we don't know if we are going to do that I think this approach is ok and we can adjust down the line if needed.